tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -1147,18 +1232,71 @@ tensorflow::Status ConvertConv2DHelper(
                                            node_def.name());
 }
 
+bool PrepareTensorForShape(Converter& ctx, const TRT_TensorOrWeights& input,
```

Comment:
Please fix the order of input and output parameters here and all other places in this PR.

Reply:
Did a git diff and fixed two other functions with the same problem :)